### PR TITLE
DOCS-697: align code sample instructions

### PR DESCRIPTION
**Pull request by sguequierre**
- subtask of DOCS-480 alignment
- remove full code samples in favor of directing to code sample tab 
- standardize component names
- standardize code samples

### CODE REVIEW

File: docs/components/movement-sensor/_index.md [hunk 1]
```diff
@@ -612,18 +560,16 @@ readings = await my_sensor.get_readings()
 **Returns:**
 
 - (map[[string]](https://pkg.go.dev/builtin#string)interface{}): A map containing the measurements from the sensor.
-  Contents depend on sensor model and can be of any type.
+Contents depend on sensor model and can be of any type.
```
Comment (JessamyT) [inline docs/components/movement-sensor/_index.md:563]:
```suggestion
    Contents depend on sensor model and can be of any type.
```
I've been indenting these to make it clear that it's meant to be part of the bullet

File: docs/components/movement-sensor/_index.md [hunk 2]
```diff
@@ -683,7 +630,7 @@ For more information, see the [Go SDK Code](https://github.com/viamrobotics/rdk/
 
 ## Next Steps
 
-Try adding a movement sensor to your [mobile robot](../base/) and writing some code with our [SDKs](../../program/sdk-as-client/) to implement closed-loop movement control for your robot.
+Try adding a movement sensor to your [mobile robot](/components/base) and writing some code with our [SDKs](/program/sdk-as-client) to implement closed-loop movement control for your robot.
```
Comment (JessamyT) [inline docs/components/movement-sensor/_index.md:633]: Why this change? I thought we are doing relative links now so that they work in the PR builds?

File: docs/components/servo.md [hunk 3]
```diff
@@ -367,8 +286,8 @@ Stop the servo from moving.
 
 For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/components/servo/client/index.html#viam.components.servo.client.ServoClient.stop).
 
-```python {class="line-numbers linkable-line-numbers"}
-my_servo = Servo.from_robot(robot=robot, name='my_servo')
+```python {class="line-numbers l inkable-line-numbers"}
```
Comment (JessamyT) [inline docs/components/servo.md:289]:
```suggestion
```python {class="line-numbers linkable-line-numbers"}
```

File: docs/components/movement-sensor/_index.md (hunk 2 above)
Comment (sguequierre) [inline docs/components/movement-sensor/_index.md:633]:
```suggestion
Try adding a movement sensor to your [mobile robot](../base) and writing some code with our [SDKs](../../program/sdk-as-client) to implement closed-loop movement control for your robot.
```

Comment (sguequierre) [reply docs/components/movement-sensor/_index.md]: Good catch changed :)

**Review by JessamyT — COMMENTED** [review]
I was told at one point to use single quotes around short symbol-like strings with Python

**Comment by sguequierre** [comment]
> I was told at one point to use single quotes around short symbol-like strings with Python

It doesn't really matter in Python, so don't need to make this change necessarily, but I think it's more common to use the double quotes + it matches the go samples

File: docs/components/input-controller/_index.md [hunk 4]
```diff
@@ -44,281 +44,475 @@ Once you've configured your input controller according to model type, you can wr
...
+- `events` [(Dict[Control, Event])](https://docs.python.org/3/library/typing.html#typing.Dict): A dictionary mapping the most recent Event for each Control.
 
-    // Wait to exit mainWithArgs() until Context is Done.
-    <-ctx.Done()
+For more information, see the [Python SDK Docs](https://python.viam.dev/_modules/viam/components/input/input.html#Controller.get_events).
 
-    return nil
+```python {class="line-numbers linkable-line-numbers"}
+# Get the most recent Event for each Control.
+recent_events = await myController.get_events()
```
Comment (JessamyT) [inline docs/components/input-controller/_index.md:184]: Should this and other ones be
```suggestion
recent_events = await my_controller.get_events()
```
?
Or is it my_input_controller since sometimes it's that way below? Sorry if I'm confused

File: docs/components/input-controller/_index.md [hunk 5]
```diff
@@ -44,281 +44,475 @@ Once you've configured your input controller according to model type, you can wr
...
+- `events` [(map[Control]Event)](https://pkg.go.dev/go.viam.com/rdk/components/input#Control): A map mapping the most recent Event for each Control.
+- `error` [(error)](https://pkg.go.dev/builtin#error): An error, if one occurred.
 
-### Event Object
+For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/components/input#Controller).
 
-Each `Event` object represents a singular event from the input device, and has four fields:
+```go {class="line-numbers linkable-line-numbers"}
+// Get the most recent Event for each Control.
+recent_events, err := myController.Events(context.Background(), nil)
```
Comment (JessamyT) [inline docs/components/input-controller/_index.md:207]:
```suggestion
recent_events, err := my_controller.Events(context.Background(), nil)
```

File: docs/components/input-controller/_index.md [hunk 6]
```diff
@@ -44,281 +44,475 @@ Once you've configured your input controller according to model type, you can wr
...
+- `error` [(error)](https://pkg.go.dev/builtin#error): An error, if one occurred.
 
-// Key release.
-ButtonRelease EventType = "ButtonRelease"
+For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/components/input#Controller).
 
-// Key is held down. This will likely be a repeated event.
-ButtonHold EventType = "ButtonHold"
+```go {class="line-numbers linkable-line-numbers"}
+myController, err := controller.FromRobot(myRobotWithController, "my_controller")
```
Comment (JessamyT) [inline docs/components/input-controller/_index.md:265]: why is it myController instead of myInputController if movement sensor is being expanded from my_sensor to my_movement_sensor in the other doc?

File: docs/components/movement-sensor/_index.md [hunk 7]
```diff
@@ -40,57 +40,13 @@ Model | Supported hardware <a name="model-table"></a>
...
-linAccel, err := myIMU.LinearAcceleration(context.Background(), map[string]interface{}{})
-if err != nil { 
-  logger.Fatalf("cannot get linear acceleration: %v", err) 
-}
-logger.Info("my-imu LinearAcceleration in the x direction: ", linAccel.X)
-```
-
-{{% /tab %}}
-{{< /tabs >}}
+These examples assume you have a movement sensor called `"my_movement_sensor"` configured as a component of your robot.
```
Comment (JessamyT) [inline docs/components/movement-sensor/_index.md:48]: Good call changing it from my_imu but it could stay my_sensor for brevity if we're not rigidily tied to the full component name (as it looks like we're not in input controller?)

**Review by JessamyT — COMMENTED** [review]
Looks about good but I'm a little lost with all the different names of input controller so want to double check that that's all as intended

File: docs/components/input-controller/_index.md (hunk 5 above)
Comment (sguequierre) [reply docs/components/input-controller/_index.md:207]: This is just go convention vs python convention here!

File: docs/components/input-controller/_index.md (hunk 4 above)
Comment (sguequierre) [reply docs/components/input-controller/_index.md:184]: Yeah, thanks for pointing this out -- I realized this one didn't have the `my_controller = Controller.from_robot(robot=robot, name="my_controller")` line etc so am fixing that~~ will switch to Python convention on this line too

Comment (sguequierre) [reply docs/components/input-controller/_index.md]: Fixed!

File: docs/components/movement-sensor/_index.md [hunk 8]
```diff
@@ -587,18 +537,16 @@ If a sensor is not configured to take a certain measurement or fails to read a p
 
 **Returns:**
 
-- ([Mapping](https://docs.python.org/3/glossary.html#term-mapping)[[str](https://docs.python.org/3/library/stdtypes.html#str), Any]): An object containing the measurements from the sensor.
-  Contents depend on sensor model and can be of any type.
+- ([Mapping [str, Any]](https://docs.python.org/3/glossary.html#term-mapping)): An object containing the measurements from the sensor.
```
[the remark this answers is not in the document]
Comment (sguequierre) [reply docs/components/movement-sensor/_index.md:540]: If anything on this one I feel like Any would be the most helpful, but I think with this complex typing sometimes it should be fine to link to the top type right?

File: docs/components/movement-sensor/_index.md (hunk 7 above)
Comment (sguequierre) [reply docs/components/movement-sensor/_index.md:48]: Do you think it could be confusing with the sensor code samples / if they had a sensor configured separately and used code sample tab?

File: docs/components/input-controller/_index.md (hunk 6 above)
Comment (JessamyT) [reply docs/components/input-controller/_index.md]: okay fine as is then!

File: docs/components/movement-sensor/_index.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (JessamyT) [reply docs/components/movement-sensor/_index.md:540]: I try to link to all 3 where applicable but yeah not a big deal and if you think this is better I don't feel strongly

**Review by JessamyT — APPROVED** [review]
